__tests__/core/runner.test.ts

```diff
@@ -793,4 +793,144 @@ describe('runner', () => {
       });
     });
   });
+
+  describe.only('journey and step annotations', () => {
+    const getBufferData = () => {
+      const fd = fs.openSync(dest, 'r');
+      const buffer = fs.readFileSync(fd, 'utf-8');
+      // get as string and remove the last line
+      return buffer.split('\n').slice(0, -1).join('\n');
+    };
+
+    it('skip journey', async () => {
+      runner.addJourney(
+        journey.skip('j1', async () => {
+          step('step1', async () => {});
+        })
+      );
+      runner.addJourney(
+        journey('j2', async () => {
+          step('step1', async () => {});
+        })
+      );
+      await runner.run(defaultRunOptions);
+      const data = getBufferData();
```

Comment:
Would be better to check explicity which ran and which skipped - We could use `readAndCloseStreamJson` and check the json response.